### ISSUE TITLE
Statistics on a specific room

### ISSUE THREAD

**Issue by ValentinValETS** [mozilla/hubs#4406]
Hi everybody,

I'm looking to have simple statistics data on a Hubs room : number of connections since the room's creation, the time they spent for each connection, etc...

Is there a simple way to do this or is this a new feature that could be included in a future version ?

Thanks !

**Comment by matthewbcool** [comment]
Hi Valentin! Not currently.

You may find using the discord bot creatively can help you with room statistics:  https://hubs.mozilla.com/docs/hubs-discord-bot.html

You can customize analytics much further by building them into your own client using hubs cloud: https://hubs.mozilla.com/docs/hubs-cloud-intro.html

**Comment by teo523** [comment]
Hi Valentin, 
I also ran into these issues lately. I found the bots really helpful for doing this. I actually created an example where I extract the position of every avatar and map it into a js script to create a 2D map of the room. Check it out [here](https://metapoiesis.herokuapp.com/).
The code for this is available in my repo, check [this example](https://github.com/teo523/hubs-client-bot/blob/master/examples/export-avatars-positions.js) and let me know if you have any doubts!
